pygments/lexers/tal.py

```diff
@@ -0,0 +1,58 @@
+"""
+    pygments.lexers.uxntal
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for Uxntal
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+import keyword
+
+from pygments.lexer import RegexLexer
+from pygments.token import Comment, Keyword, Name, String, Number, Punctuation, Whitespace, Literal
+
+__all__ = ['TalLexer']
+
+class TalLexer(RegexLexer):
+
+    name = 'Tal'
+    aliases = ['tal', 'uxntal']
+    filenames = ['*.tal']
+    mimetypes = ['text/x-uxntal']
+
+    tokens = {
+        'comment': [
+            (r'(?:(?<=\s)|^)\((?:(?=\s)|$)', Comment.Multiline, '#push'), # nested comments
```

Comment:
You can simplify this regex to `r'(?<!\S)\((?!\S)`. Same on the line below.

Reply:
That's a good point. I'll go through a simplify a bunch of these using this pattern.